content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -136,7 +132,7 @@ const result = await prisma.$queryRaw<User[]>`SELECT * FROM User`
 
 > **Note**: If you do not provide a type, `$queryRaw` defaults to `unknown`.
 
-If you are selecting **specific fields** of the model or want to include relations, refer to the documentation about [leveraging Prisma Client's generated types](/orm/prisma-client/type-safety) if you want to make sure that the results are properly typed.
+If you are selecting **specific fields** of the model or want to include relations, refer to the documentation about [leveraging Prisma Client's generated types](/orm/prisma-client/type-safety/operating-against-partial-structures-of-model-types#problem-using-variations-of-the-generated-model-type) if you want to make sure that the results are properly typed.
```

Comment:
This is the core change of the PR. The old link just went to the parent page, while this goes to the correct subpage and headline.